projects/gloo/pkg/bootstrap/vault.go

```diff
@@ -4,8 +4,19 @@ import (
 	"github.com/hashicorp/vault/api"
 	errors "github.com/rotisserie/eris"
 	v1 "github.com/solo-io/gloo/projects/gloo/pkg/api/v1"
+	"github.com/solo-io/solo-kit/pkg/api/v1/clients/factory"
 )
 
+const DefaultPathPrefix = "secret"
+
```

Comment:
nit: golangci-lint makes me suggest the following
```suggestion
// NewVaultSecretClientFactory consumes a vault client along with a set of basic configurations for retrieving info with the client
```